src/api/guardian.js

```diff
@@ -0,0 +1,17 @@
+const guardian = require("guardian-js");
+require("env2")("./config.env");
+const guardian_key = process.env.GUARDIAN_KEY;
+
+const guardianKey = (query, cb) => {
+  let api = new guardian(guardian_key, false);
```

Comment:
Why is this variable a `let` when everything else is a `const`?

Reply:
It's the way recommended from the documentation https://github.com/PorterK/GuardianJSClient

Reply:
well this documentation also has 5 stars only and almost not being used on npm so not sure i would trust it and i would change it const

**that means that you should change it to const btw**

Reply:
https://github.com/PorterK/GuardianJSClient/issues/10

happy ?